hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -503,5 +504,43 @@ public void testFlakeIDGeneratorCount() {
 
     }
 
+    @Test
+    public void testMapPutLatency() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptla"), "0");
+
+        IMap<Object, Object> iMap = node.hazelcastInstance.getMap("hazelcast");
+        iMap.put("1", "hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptla"),
+                String.valueOf(iMap.getLocalMapStats().getTotalPutLatency() / iMap.getLocalMapStats().getPutOperationCount()));
+
+        iMap.put("2", "phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptla"),
+                String.valueOf(iMap.getLocalMapStats().getTotalPutLatency() / iMap.getLocalMapStats().getPutOperationCount()));
```

Comment:
This test calculates the expected output in the same way as the implementation, which means it will never fail. Please improve the test to use concrete input and output values (there are multiple ways of doing it, but I think you can figure it out)